mc2/controllers/docker/models.py

```diff
@@ -62,11 +63,16 @@ def get_marathon_app_data(self):
         service_labels = self.get_default_app_labels()
         service_labels.update({
             "domain": domains,
```

Comment:
This is used by the consul-template+nginx setup, so it should probably move to the "external visibility" section.